packages/commonwealth/server/models/template.ts

```diff
@@ -8,6 +8,10 @@ export type TemplateAttributes = {
   abi_id: number;
   name: string;
   template: string; // TODO: Type this as an object to the schema spec??
+  created_by?: string;
```

Comment:
Does created_by ever get set?

Reply:
Its not needed in this PR, but it will be in the next fast follow so included it so the model updates were all in one commit